tests/kafkatest/tests/produce_consume_validate.py

```diff
@@ -110,7 +111,8 @@ def validate(self):
         consumed = self.consumer.messages_consumed[1]
         missing = set(acked) - set(consumed)
 
-        self.logger.info("num consumed:  %d" % len(consumed))
+        self.logger.info("Producer produced %d messages where %d messages are unique" % (len(acked), len(acked)))
```

Comment:
Why log the number twice? When I read the message, I though the two values were going to be something like `len(acked)` vs `len(set(acked))`, but the message will always just say all the messages are unique?